src/index.js

```diff
@@ -1 +1,36 @@
-import './style.css';
\ No newline at end of file
+// imports
+import './style.css';
+import ScoreLibrary from './manage_scores.js';
+import * as DOM from './dom_manipulation.js';
+// global variablesx
+const score = new ScoreLibrary();
+// score[list].forEach((scoreData) => DOM.createScore(scoreData));
+// query selectors
+const scoreForm = document.querySelector('#score_form');
+const playerName = document.querySelector('#name_input');
+const playerScore = document.querySelector('#score_input');
+const refresButton = document.querySelector('#refresh_button');
+const scoreTable = document.querySelector('#score_table');
+// functions
+
+// event listeners
+scoreForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const newScore = {
+    user: playerName.value,
+    score: playerScore.value,
+  };
+  score.send(newScore).then(() => {
+    playerScore.value = '';
+    playerName.value = '';
+  });
+});
+
+refresButton.addEventListener('click', () => {
+  scoreTable.innerHTML = '';
+  score.get().then(() => {
+    score.list.forEach((element) => {
+      DOM.createScore(element);
+    });
+  });
+});
```

Comment:
- [OPTIONAL] To enhance UX, we can extract the logic of this event listener into a separate function, say, **`displayScores`**. We can then use this function in two separate places. One, in this very event listener. Two, in a `window.onload` event listener to load the scores when the page first loads. See examples below:

```JavaScript
const displayScores = () => {
  scoreTable.innerHTML = '';
  score.get().then(() => {
    score.list.forEach((element) => {
      createScore(element);
    });
  });
};

refresButton.addEventListener('click', () => {
  displayScores();
});

window.onload = () => {
  displayScores();
};
```

Reply:
Thanks for your suggestion @Lameck1 

I'm going to implement that feature in the next milestone.